charts/kyverno/README.md

```diff
@@ -60,8 +60,28 @@ Unfortunately `kubectl` adds metadata that will cross the limit allowed by Kuber
 
 Another option is to use server side apply, this will be supported in ArgoCD v2.5.
 
-Below is an example of ArgoCD application manifest that should work with this chart:
+Finally, we introduced new CRDs in 1.8.0 to manage resource level reports, those reports are attached to their underlying resource using owner reference.
+
+ArgoCD will show those reports in the UI and as they are managed dynamically by Kyverno it can pollute your dashboard.
```

Comment:
I don't quite understand what you're trying to say in this sentence.

Reply:
Reports will show up in argocd UI (because reports are attached to the resource argocd makes them visible in the ui).

Though users don't create those resources and they might be confused seeing them in argocd.